modules/mono/editor/GodotTools/GodotTools/Internals/Internal.cs

```diff
@@ -17,6 +17,13 @@ internal static partial class Internal
         public const string CSharpLanguageType = "CSharpScript";
         public const string CSharpLanguageExtension = ".cs";
 
+        public enum PopupSeverity
+        {
+            SEVERITY_INFO = 0,
+            SEVERITY_WARNING,
+            SEVERITY_ERROR,
+        };
```

Comment:
- Enums in Godot are marshalled as `long`.
- We should try to match the name of the enum in Core, since the enum is inside the `EditorToaster` type I suggest changing the prefix to match that.
- .NET naming conventions are to use PascalCase for enum members and avoid using the type name as a prefix.

```suggestion
        public enum EditorToasterSeverity : long
        {
            Info = 0,
            Warning = 1,
            Error = 2,
        };
```